src/contract.rs

```diff
@@ -91,56 +99,52 @@ fn try_write_result<S: Storage, A: Api, Q: Querier>(
         verified,
     };
 
-    save(&mut deps.storage, specific_key.as_bytes(), &result)?;
-    save(&mut deps.storage, verified_key.as_bytes(), &verified)?;
+    save(deps.storage, specific_key.as_bytes(), &result)?;
+    save(deps.storage, verified_key.as_bytes(), &verified)?;
 
     if verified {
-        let mut verified_ids: Vec<u16> =
-            may_load(&deps.storage, "all_verified".as_bytes())?.unwrap_or_default();
+        let mut verified_ids: Vec<u32> =
```

Comment:
Honestly I set it to a u16 because I felt like there since there aren't many contracts right now, 65k code IDs would be fine for a long time. But I guess u32 is more future proof.

Reply:
oh yeah, it might be a while til they get to that point, but you never know!